services/admin/src/store.ts

```diff
@@ -296,21 +343,22 @@ export class Store {
             continue;
           }
 
-          // Parse CVR data for validation.
-          const cvr = safeParseJson(line).unsafeUnwrap() as CastVoteRecord;
+          const [cvrParseResult] = [...parseCvrs(line, election)];
```

Comment:
Do we ever want to call `parseCvrs` with more than one line? Maybe simplify that function since we're doing the line streaming here. Alternatively, use an iterable-friendly approach?
```suggestion
          const cvrParseResult = take(1, parseCvrs(line, election));
```

Reply:
yeah, definitely opportunities for drastic simplification here - at the moment, we're in the in-between where it's being used both client and server-side, so post-refactoring, we can get surgical with `parseCvrs` and other older parts of the code. not even a fan of the fact that it conflates parsing with data validation, but that's a gripe for another day 🙃

Reply:
I would like to hear more about what you think a better model for parsing/data validation would be. Are you primarily talking about splitting the JSON parsing from the object validation, or something different?

Reply:
yeah, thinking along the lines of keeping things modular and separating the concerns of parsing a CVR object from a raw string vs validating that the values are valid for a given election.
contrived case: `parseCvrs` doesn't easily lend itself to validating an existing CVR object, for example, without first stringifying